bufferserver/src/test/java/com/datatorrent/bufferserver/util/CodecTest.java

```diff
@@ -22,8 +22,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testng.annotations.Test;
-import static org.testng.Assert.*;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
Not sure if we allow import static.  Should we just import org.junit.Assert and call Assert.assertEquals?